backend/apps/ifc_validation/tasks.py

```diff
@@ -716,16 +728,91 @@ def is_schema_error(line):
 @shared_task(bind=True)
 @log_execution
 @requires_django_user_context
-def bsdd_validation_subtask(self, prev_result, id, file_name, *args, **kwargs):
+@update_progress
+def digital_signatures_subtask(self, prev_result, id, file_name, *args, **kwargs):
 
     # fetch request info
     request = ValidationRequest.objects.get(pk=id)
     file_path = get_absolute_file_path(request.file.name)
 
-    # increment overall progress
-    PROGRESS_INCREMENT = 10
-    request.progress = min(request.progress + PROGRESS_INCREMENT, 100)
-    request.save()
+    # add task
+    task = ValidationTask.objects.create(request=request, type=ValidationTask.Type.DIGITAL_SIGNATURES)
+
+    prev_result_succeeded = prev_result is not None and prev_result['is_valid'] is True
+    if prev_result_succeeded:
+
+        task.mark_as_initiated()
+
+        # determine program/script to run
+        check_script = os.path.join(os.path.dirname(__file__), "checks", "signatures", "check_signatures.py")
+        check_program = [sys.executable, check_script, file_path]
+        logger.debug(f'Command for {self.__qualname__}: {" ".join(check_program)}')
+
+        # check schema
```

Comment:
```suggestion
        # check signatures
```

Possibly a holdover from the bSDD task?